src/test/java/se/nackademin/HatTest.java

```diff
@@ -0,0 +1,78 @@
+package se.nackademin;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class HatTest {
+    
+
+    @Test
+    public void assertNamenotNull(){
+        Hat ahat = new Hat("Slick top hat", "Fancy hat company", "XL", "purple", 
```

Comment:
While unit tests should be doing mostly one thing at a time, in this case I would opt for creating the Hat once and then just checking that all of it's attributes are not null.